presto-docs/src/main/sphinx/connector/delta.rst

```diff
@@ -0,0 +1,96 @@
+provided by Delta Lake project to read the table metadata.
+
+Configuration
+-------------
+
+To configure the Delta Lake connector, create a catalog properties file
+``etc/catalog/delta.properties`` with the following contents,
+replacing the properties as appropriate:
+
+.. code-block:: none
+
+    connector.name=delta
+    hive.metastore.uri=hostname:port
+
+Configuration Properties
+------------------------
+
+The following configuration properties are available:
+
+=============================================== ========================================================= ============
+Property Name                                   Description                                               Default
+=============================================== ========================================================= ============
+``hive.metastore.uri``                          The URI(s) of the Hive metastore where Delta Lake tables  ``null``
+                                                are registered. The connector relies on the Hive
+                                                metastore to find the location of Delta Lake tables.
+                                                From the Delta Log at given location, schema and data
+                                                file list of the table is found.
+``delta.partition-pruning-enabled``             To enable partition pruning of files based on the filter  ``true``
+                                                in query.
+``delta.filter-pushdown-enabled``               To enable pushing the filter into Delta table scan to     ``true``
+                                                prune the list of files to read and use the filter to
+                                                data skipping in data file reader (e.g. Row group
+                                                skipping in Parquet reader).
+``delta.parquet-dereference-pushdown-enabled``  Enable pushing nested column dereferences into            ``true``
+                                                table scan so that only the required fields
+                                                selected in a ``struct`` data type column are selected.
+                                                In order for this option to work, also set
+                                                ``experimental.pushdown-dereference-enabled`` to
+                                                ``true``.
+=============================================== ========================================================= ============
+
+Delta Lake connector reuses many of the modules existing in Hive connector.
+Modules for connectivity and security such as S3, Azure Data Lake, Glue metastore etc.
+So the configurations for these modules is same those available in Hive connector documentation.
+
+Querying Delta Lake Tables
+--------------------------
+Example query
```

Comment:
Can you also provide a CREATE TABLE example? I see from tests that we don't need to provide full schema when creating a table. Will be helpful if we mention that we need to provide a dummy column in DDL and also that partition specification is not required.

Reply:
Thats a good idea. Added to the documentation.